.travis.yml

```diff
@@ -15,12 +22,13 @@ before_deploy:
 - docker login --username="$DOCKER_USERNAME" --password=$(heroku auth:token) registry.heroku.com
 - docker pull $DOCKER_USERNAME/my-production-image:latest
 - docker tag  $DOCKER_USERNAME/my-production-image:latest registry.heroku.com/trello-to-do/web
-
+# - docker push registry.heroku.com/trello-to-do/web
+- terraform apply -var="TODOBOARD=$TODOBOARD" -var="DOINGBOARD=$DOINGBOARD" -var="DONEBOARD=$DONEBOARD" -var="ARM_CLIENT_ID=$ARM_CLIENT_ID" -var="ARM_SUBSCRIPTION_ID=$ARM_SUBSCRIPTION_ID" -var="ARM_CLIENT_SECRET=$ARM_CLIENT_SECRET" -auto-approve
```

Comment:
Once you've removed some of the sensitive variables defaults (see below) then you'll have some additional variables to pass through here